src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify later conditions. Also, don't add `else` after `if` with `return` - the code after it won't be executed anyway. So the code can be simplified to: js if (days >= LONG_TERM_RENTAL) { return price - LONG_TERM_DISCOUNT; } if (days >= SHORT_TERM_RENTAL) { return price - SHORT_TERM_DISCOUNT; } return price; ``` This way, as soon as a condition is met, the function will return and stop executing further code.